components/edit-collective/sections/HostVirtualCards.js

```diff
@@ -354,7 +357,8 @@ const HostVirtualCards = props => {
             {...vc}
             onSuccess={refetch}
             editHandler={() => setEditingVirtualCard(vc)}
-            hasActions
+            // Edit card action need to be reworked
```

Comment:
Maybe add a `TODO` to support better searching in the future, 

```suggestion
            // TODO: Edit card action need to be reworked
```